subtrace.go

```diff
@@ -27,19 +28,17 @@ func main() {
 	c.Name = filepath.Base(os.Args[0])
 	c.ShortUsage = "subtrace <command>"
 
-	c.Subcommands = append(c.Subcommands, run.NewCommand())
+	if runtime.GOOS == "linux" {
```

Comment:
Not sure if the more idiomatic way would be to put this logic in a separate file so we can use a build tag instead of an `if` condition